test/property.test.js

```diff
@@ -125,6 +292,20 @@ describe('loopback:property generator', function() {
     });
   });
 
+  it('fails to create with unsupported type', function (done) {
+    var propertyGenerator = givenPropertyGenerator();
+    helpers.mockPrompt(propertyGenerator, {
+      model: 'Car',
+      name: 'something',
+      type: 'unknown',
+      required: 'true',
+      defaultValue: 'X'
+    });
+
+    expect(propertyGenerator.run).to.throw(Error);
```

Comment:
will take it out completely as no longer relevant since we only fail for number casting and that is tested anyway.